api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -335,10 +335,16 @@
     Attribute<String> APPLICATION_DATA_LINK = new Attribute<String>(new String2String());
     /**
      * Transport to push the stats and the metrics, "builtin:{topic}" if STRAM should push the data directly
-     * using websocket with the given topic
+     * using websocket with the given topic.
+     * If using a custom transport, please use Object2String codec to specify the transport object.  The object must
+     * be from a class that implements the AutoMetric.Transport interface
      */
     Attribute<String> METRICS_TRANSPORT = new Attribute<String>(new String2String());
```

Comment:
I originally wanted to do this, but how do we support this while supporting the built-in websocket transport at the same time? Looking for suggestions on how to do this a better way.